decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -178,7 +178,7 @@ def add_queue_adapter
         route <<~RUBY
           authenticate :user, ->(u) { u.admin? } do
             mount Sidekiq::Web => "/sidekiq"
-          end      
+          end#{"      "}
```

Comment:
Could you explain a bit this change?

Reply:
This particular change was caught by `Layout/TrailingWhitespace` rubocop rule. 

As per doc, i could use something like the below snippet to fix it. 
```
# good
trailing_whitespace = ' '
code = <<~RUBY
  x = 0#{trailing_whitespace}
RUBY
```

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/TrailingWhitespace

Reply:
This seems to have been created by rubocop -a ... Reverting .